frontend/assets/.gitignore

```diff
@@ -0,0 +1,2 @@
+# dependencies
+/node_modules
```

Comment:
@xaviertactiv remove this file and move it into the .gitignore in the root directory